<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gctMainWindow</class>
 <widget class="QMainWindow" name="gctMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gctMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>148</width>
         <height>429</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="CoordMenuButton">
          <property name="text">
           <string>Coordinates</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="CoordPage">
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="CoordTabWigdet">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="CoordTab01">
           <attribute name="title">
            <string>Translate</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLineEdit" name="CoordLineDegree"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="CoordButtonDegree">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="CoordLineDegreeMinute"/>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="CoordLineDegreeMinuteSecond"/>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="CoordButtonDegreeMinute">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="CoordButtonDegreeMinuteSecond">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegree">
              <property name="text">
               <string>Degrees (e.g. 53.087117 8.791817)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegreeMinute">
              <property name="text">
               <string>Degrees Minutes (e.g. N 53째 05.227 E 008째 47.509)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QLabel" name="CoordLabelDegreeMinuteSecond">
              <property name="text">
               <string>Degrees Minutes Seconds (e.g. N53째 5' 13.62&quot; E8째 47' 30.54&quot;)</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CoordTab02">
           <attribute name="title">
            <string>Project</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>gctMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
